exercises/numbers/factors/factors.js

```diff
@@ -14,11 +14,21 @@ function factors(num) {
   let results = [];
 
   for (let i = 1; i <= num; i++) {
-    // Your code goes here. :)
+    Math.floor(Math.sqrt(num))
```

Comment:
Seems like you're just copying from https://exercism.io/tracks/javascript/exercises/legacy-prime-factors/solutions/2d35d766b68d4448a16a2730d8a6c914 ?

1. Slow down
1. Be clear about what the exercise is asking
1. Do the exercise by hand using the framework from the basic-syntax repo if need be
1. Try easier exercises first
1. Re-read the relevant sections of the crash course (in the case of `factors` that's most of it, which is why it's one of the later exercises)